src-web/components/kappnav/ComponentView.jsx

```diff
@@ -44,23 +46,45 @@ class ComponentView extends React.Component {
       sortDirection: SORT_DIRECTION_ASCENDING,
       pageSize: PAGE_SIZES.DEFAULT,
       pageNumber: 1,
-      search: undefined
+      search: undefined,
+      resourceType : (location.pathname.split('/').filter(function(e){return e})[1] === 'applications')? RESOURCE_TYPES.APPLICATION : (location.pathname.split('/').filter(function(e){return e})[1] === 'was-nd-cells')? RESOURCE_TYPES.WASNDCELL : RESOURCE_TYPES.LIBERTYCOLLECTIVE,
+      name : decodeURIComponent(location.pathname.split('/').filter(function (e) { return e })[2])
```

Comment:
We should also refactor this code.  May places we are trying to tokenize the URL path.  We should just make this logic common rather than copy/pasting `location.pathname.split('/').filter(function (e) { return e }` in multiple places in this file.